client.proto

```diff
@@ -194,11 +194,46 @@ message ReplToken {
   Permissions permissions = 15;
 }
 
-// GovalTokenMetadata is information about a goval token, that can be used to
+// GovalSigningAuthority is information about a goval token, that can be used to
 // validate it. It is stored in the footer of the PASETO.
-message GovalTokenMetadata {
-  // The ID of the key that was used to sign the token.
-  string key_id = 1;
+message GovalSigningAuthority {
+  oneof cert {
+    // The ID of the public key that was used to sign the token.
+    string key_id = 1;
+
+    // The intermediate_cert that was used to sign the token.
+    string intermediate_cert = 2;
+  }
+}
+
+enum FlagClaim {
+	MINT_GOVAL_TOKEN = 0;
+	MINT_INTERMEDIATE_TOKEN = 1;
+}
+
+message CertificateClaim {
+	oneof claim {
+		string replid = 1;
+		string user = 2;
+		FlagClaim flag = 3;
+	}
+}
+
+message GovalCert {
```

Comment:
Can this also have some documentation? a similar level of detail to `ReplToken` would be appreciated. Notably, this should mention that this goes in the body of the PASETO.